### DISCUSSION TITLE
why do the images available for download not have the new FAT partition for configuration

### DISCUSSION THREAD

**Discussion by dirkhh** [MichaIng/DietPi#6643] [Q&A] [answered]
Admittedly, I only checked a couple, but I had assumed that all of the new images from last weekend would have the trailing FAT partition... that was kinda the goal of that addition. But so far for the ones where I looked, that wasn't the case.
I this something that gets disabled in the build process for the official images? And may I ask why?

**Comment by MichaIng** [comment, marked as answer]
The partition should be present in all images aside of x86_64 installer, container and VM ones. Did you download the new xz-compressed images or the old 7z ones? The old ones are only still there to give vendors some time to change their links, but the download page and docs all link to the xz ones already.
